src/main/java/com/es/phoneshop/model/product/ProductService.java

```diff
@@ -7,8 +7,13 @@
 import java.util.stream.Collectors;
 
 public class ProductService {
+    private static ProductService instance = new ProductService();
     private final String QUERY = "";
```

Comment:
assign a good name, `MATCH_ALL_QUERY` or sth like that

Reply:
did it